src/main/resources/org/sonatype/nexus/ci/iq/Messages.properties

```diff
@@ -30,6 +30,7 @@ IqPolicyEvaluation.FailOnNetwork=Fail build when unable to communicate with IQ S
 IqPolicyEvaluation.JobSpecificCredentials=Use job specific credentials
 
 IqPolicyEvaluation.NoIqServersConfigured=No IQ Server configured.
+IqPolicyEvaluation.ResourceNotFound=Possible version mismatch.  Please check if the IQ Server is compatible with this plugin.
```

Comment:
If we're going to use this for all 404s, I might extend the message a bit, such as:

`Required resource not found, possible version mismatch. Please check if the IQ Server is reachable and compatible with this plugin.`

At least that way we cover both options, I'd hate for something basic like an unreachable server because of some internal network weirdness to have someone spend a day thinking they have the wrong version. I know they should check such things, but if you're a techie in a pressure-cooker environment, I think it's at least possible they'll take the error message for more than they should, and it could lead them astray.

Reply:
> If we're going to use this for all 404s, I might extend the message a bit, such as:

Changed message to be specific e299be4